README.md

```diff
@@ -1,24 +1,129 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+# 📖 [My Blog] <a name="about-project"></a>
 
-* System dependencies
+**My Blog** is a classic example of a blog website. This is a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
 
-* Configuration
+## 🛠 Built With <a name="built-with"></a>
 
-* Database creation
+Ruby, Ruby on Rails, Visual Studia Code, PostgreSQL
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+<details>
+  <summary>Ruby on Rails</summary>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>PostgreSQL</summary>
+</details>
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- **[Database Connection and ORM]**
+- **[Authentication ]**
+- **[authorization ]**
+- **[MVC architecture ]**
+- **[Active Record Migrations]**
+- **[User-Friendly Interface]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Setup <a name="setup"></a>
+
+In order to use this project.. Clone this repository to your desired folder by pasting this command in your command line interface:
+
+https://github.com/newtonah0/My-Blog1.git
+
+### Prerequisites <a name="prerequisites"></a>
+
+Ruby
+Ruby on Rails
+PostgreSQL
+Git
+
+### Install <a name="install"></a>
+
+To install the project's dependencies, run:
+
+```
+bundle install
+```
+
+### Usage <a name="usage"></a>
+
+To use, run:
+
+```
+rails server
+or
+rails s
+```
+
+### Run tests <a name="run tests"></a>
+
+To test, run:
+
+```
+rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="author"></a>
+
+- GitHub: [@newtonah0](https://github.com/newtonah0)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Mobile App]**
+- [ ] **[User Profiles]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/newtonah0/My-Blog/issues).
```

Comment:
- [ ] Please make sure that you don't have any broken links in your `README` file because it has a bad impact on the project contributors and viewers to struggle in order to write an issue or suggestion. I checked it and I got the error page `404` and reason behind the error is name of the repository in your link is incorrect. Kindly fiix the issue and submit project again.